src/element.ts

```diff
@@ -225,4 +225,168 @@ export abstract class Element {
   checkContext(): RenderContext {
     return defined(this.context, 'NoContext', 'No rendering context attached to instance.');
   }
+
+  //////////////////////////////////////////////////////////////////////////////////////////////////
+  // Font Handling
+
+  /**
+   * Provide a CSS compatible font string (e.g., 'bold 16px Arial').
+   */
+  set font(f: string) {
+    this.setFont(f);
+  }
+
+  /** Returns the CSS compatible font string. */
+  get font(): string {
+    return Font.toCSSString(this.textFont);
+  }
+
+  /**
+   * Set the element's font family, size, weight, style (e.g., `Arial`, `10pt`, `bold`, `italic`).
+   * @param font is 1) a `FontInfo` object or
+   *                2) a string formatted as CSS font shorthand (e.g., 'bold 10pt Arial') or
+   *                3) a string representing the font family (at least one of `size`, `weight`, or `style` must also be provided).
+   * @param size a string specifying the font size and unit (e.g., '16pt'), or a number (the unit is assumed to be 'pt').
+   * @param weight is a string (e.g., 'bold', 'normal') or a number (100, 200, ... 900).
+   * @param style is a string (e.g., 'italic', 'normal').
+   * If no arguments are provided, then the font is set to the default font.
+   * Each Element subclass may specify its own default by overriding the static `TEXT_FONT` property.
+   */
+  setFont(font?: string | FontInfo, size?: string | number, weight?: string | number, style?: string): this {
```

Comment:
This is the setFont() API, which consolidates the different ways different elements were setting their own text fonts. I tried to make it as consistent with CSS standards as possible. Even the RenderContexts (CanvasContext & SVGContext) share this `setFont(...)` method signature now.